datasets/us/de/med_exclusions/us_de_med_exclusions.yaml

```
@@ -0,0 +1,80 @@
+title: US Delaware Medicaid Sanctioned Providers
+entry_point: crawler.py
+prefix: us-medde
+coverage:
+  frequency: daily
+  start: "2024-11-27"
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+ci_test: false
+summary: >
+  List of providers excluded from participating in the Delaware Medicaid program.
+description: |
+  This dataset provides details on health care providers who have been sanctioned
+  by Medicaid or the Office of Inspector General (OIG). It includes records of
+  both individual providers and entities, outlining their sanctions, descriptions
+  of violations, and associated regulatory actions. 
```

I get the idea that Medicaid isn't an organisation in its own right - it's legislation defining services that can be reimbursed, and each state has its own authorities who administer it.

```suggestion
  This dataset provides details on health care providers who have been excluded from
  participation in the Delaware Medicaid program. It lists provider names, identifiers
  such as NPI and License numbers, as well as the authority, duration, and nature of
  the sanction.
```